addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -1,223 +1,265 @@
-$gold: #ffcc00;
+$gold: #eca801;
 $silver: #cccccc;
 $bronze: #eea91e;
 
 .website_forum {
+    .o_forum_ask_btn {
+        @include media-breakpoint-up(md) {
+            box-shadow: $box-shadow;
+            width: 200px;
+        }
+    }
+
+    .o_wforum_nav.nav.nav-pills .nav-link {
+        line-height: 1;
+        @include o-hover-text-color($body-color, $link-color);
+
+        .fa {
+            opacity: 0.5;
+        }
 
-    .question-block .question-name {
-        max-width: 80%;
+        &:hover, &.active {
+            .fa {
+                opacity: 1;
+            }
+        }
+
+        &.active {
+            background-color: rgba(theme-color('info'), 0.1);
+            color: darken(theme-color('info'), 15%)!important;
+        }
     }
 
-    .o_post_content {
+    // Single Post
+    .o_wforum_post, .note-editable.card-block {
+        word-wrap: break-word;
+
         pre {
             padding: 1.5rem;
             border-radius: 5px;
             color: $white;
             background-color: $gray-700;
         }
+
         blockquote {
             position: relative;
-            padding-left: 3rem;
-            border-left: 2px solid $gray-500;
-            font-style: italic;
-            font-weight: lighter;
+            padding-left: 1em;
+            border-left: .25em solid $gray-500;
+            color: $gray-500;
         }
-        word-wrap: break-word;
     }
 
-    .vote {
-        min-width: 38px;
-        margin-right: 12px;
-        .vote_count {
-            font-weight: bold;
-            line-height: 12px;
-            font-size: 160%;
-        }
-        .vote_down,
-        .vote_up {
-            font-size: 130%;
-        }
-    }
+    .o_wforum_readable {
+        max-width: 700px;
 
-    .post_to_validate {
-        overflow: hidden;
-        .validate_answer {
+        > p:empty:first-child:last-child {
             display: none;
         }
-    }
 
-    .validate_answer {
-        @include o-position-absolute(0, auto, 0, 0);
-        width: 4px;
-        height: 100%;
-        z-index: 1; // On top of card content if an alert is displayed inside
+        p {
+            margin-bottom: 0.5rem;
 
-        > .bg-success {
-            @include o-position-absolute(0, 0, 0, 0);
+            > br:first-child:last-child {
```

Comment:
Each time you create (or even just Edit) a post, an empty `<p><br/></p>` is added to the content.
I initially tried to remove it directly in the JS ( remember [this](https://github.com/odoo/odoo/pull/35749#discussion_r315073741)? ). This is the CSS workaround.

Reply:
If the `<br/>` is needed for the editor to work, hiding it in css will probably break it for the same reason... and indeed the text cursor does not appear on my side because of this. I don't understand what this is is fixing, could you screenshot the problem ?

Reply:
Note,  I just replaced the selector with a direct descendent to avoid issues in edition

```css
.o_wforum_readable {
        max-width: 700px;

        > p > br:first-child:last-child {
            display: none;
        }
    }
```
Screenshoot of the issue

![Screenshot 2019-09-04 at 11 01 51](https://user-images.githubusercontent.com/8558351/64249060-1aa72280-cf13-11e9-9e94-f5207af0d3e9.png)

Reply:
Mmh, you only get the problem if the user explicitly tap "enter" multiple times at the end. This should not be handled in css in forum nor in JS. It should be the "save" that remove those if needed, this is an editor problem. I'd say we remove this and keep the problem in mind (cc @JKE-be )

Reply:
maybe we should have a strip method in editor, and call it explicitly for forum.
But I confirm that do it in css is probably not the good hack. Let fix that later... thx